corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ontonotes/utils/SRLNode.java

```diff
@@ -0,0 +1,182 @@
+package edu.illinois.cs.cogcomp.nlp.corpusreaders.ontonotes.utils;
+
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.StringTokenizer;
+
+import edu.illinois.cs.cogcomp.core.datastructures.IntPair;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Constituent;
+import edu.illinois.cs.cogcomp.core.datastructures.trees.Tree;
+
+/**
+ * base class for all SRL node arguments.
+ * @author redman
```

Comment:
Can you clarify the documentation for this too? Is it suppose to represent one single constituent?

Reply:
This class represents one entry in the propbank file. For a particular verb, it contains the relationships of other bits of the sentence to that verb.